src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java

```diff
@@ -299,11 +292,8 @@ private String validateRegisterationPayload(UserRegistrationInfo userRegInfo) {
 			str.setLength(0);
 			str.append("Invalid email id");
 		}
-		if(StringUtils.isNotBlank(userRegInfo.getPhone()) && !isValidPhoneNumber(userRegInfo.getPhone())) {
-			str.setLength(0);
-			str.append("Invalid phone number");
-		}
```

Comment:
why this condition is removed - this is going to affect direct user registration flow.